styles.css

```diff
@@ -388,10 +395,365 @@
+    display: none;
+  }
+
+  .image7 {
+    display: none;
+  }
+
+  .work-desktop {
+    display: none;
+  }
+
+  .Project-Desktop {
+    display: none;
+  }
+}
+
+/* Desktop version */
+
+@media (min-width: 768px) {
+  #header {
+    border-bottom: 1px solid #dddfe6;
+    height: 6.25rem;
+  }
+
+  .Menu {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    height: 6.25rem;
+  }
+
+  .portfolio-links {
+    list-style: none;
+    display: flex;
+  }
+
+  .Menu-line {
+    display: none;
+  }
+
+  .nav-link a {
+    text-decoration: none;
+    padding: 1rem;
+    color: #1a2236;
+  }
+
+  .image1 {
+    display: none;
+  }
+
+  .image2 {
+    display: none;
+  }
+
+  .image3 {
+    display: none;
+  }
+
+  .image4 {
+    display: none;
+  }
+
+  .image5 {
+    display: none;
+  }
+
+  .Project-Mobile {
+    display: none;
+  }
+
+  #headline {
+    display: flex;
+  }
+
+  .Intro {
+    width: 50%;
+    position: relative;
+    height: 100%;
+    padding-top: 200px;
+  }
+
+  .headline-pic {
+    position: relative;
+    width: 50%;
+    background-image: url(images/Thur.png);
+    height: 54.813rem;
+    margin-bottom: 13rem;
+  }
+
+  h1 {
+    font-size: 110px;
+    line-height: 110px;
+  }
+
+  .Collaboration {
+    width: 15.875rem;
+  }
+
+  /* Desktop works */
+  .Project-Desktop {
+    display: block;
+  }
+
+  .Projects {
+    position: relative;
+    display: inline-block;
+    font-size: 110px;
+    line-height: 110px;
+    transform: rotate(-90deg);
+  }
+
+  .project-pictures {
+    position: relative;
+    margin-right: 0;
+    display: grid;
+    grid-template-columns: 33% 10rem auto;
+    grid-template-rows: 100%;
+  }
+
+  .projects button {
+    background-color: white;
+    color: #1a2236;
+    width: 11.125rem;
+    height: 2.75rem;
+    line-height: 1.5rem;
+    font-family: "Poppins", sans-serif;
+    font-weight: 500;
+    font-size: 18px;
+    border: 1px solid #1a2236;
+  }
+
+  .big-laptop {
+    width: 100%;
+    background-color: #1a2236;
+    height: 25rem;
+  }
+
+  .big-laptop img {
+    width: 100%;
+    height: 13rem;
+  }
+
+  .work {
+    display: none;
+  }
+
+  .work-desktop {
+    display: block;
+  }
+
+  .grid-container1 {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto;
+  }
+
+  .grid1 {
+    height: 16rem;
+    width: 29rem;
+    position: relative;
+    bottom: -6.5rem;
+  }
+
+  .grid2 {
+    width: 32.5rem;
+  }
+
+  .grid3 {
+    padding-left: 1.25rem;
+    background-color: #1a2236;
+    width: 100%;
+    position: relative;
+    bottom: -3.25rem;
+  }
+
+  .grid3 img {
+    position: relative;
+    width: 100%;
+    height: 80%;
+    bottom: -2.5rem;
+  }
+
+  .grid3 button {
+    background-color: white;
+    color: #1a2236;
+    width: 11.125rem;
+    height: 2.75rem;
+    line-height: 1.5rem;
+    font-family: "Poppins", sans-serif;
+    font-weight: 500;
+    font-size: 18px;
+    border: 1px solid #1a2236;
```

Comment:
- [ ] The spacing between the elements in this section is different from the Figma design, kindly stick as much as possible to the template.

![rev3](https://user-images.githubusercontent.com/53347966/230517109-35f2d020-128c-4a83-81d9-c0128a85fab1.JPG)